src/sentry/static/sentry/app/components/assistant/guideAnchor.tsx

```diff
@@ -214,8 +214,8 @@ type WrapperProps = {disabled?: boolean} & Props;
 export default class GuideAnchorWrapper extends React.Component<WrapperProps> {
   render() {
     const {disabled, children, ...rest} = this.props;
-    if (disabled) {
-      return children;
+    if (disabled || window.localStorage.getItem('hide_anchors') === '1') {
+      return children || null;
```

Comment:
Why does this need `null`?

Reply:
@billyvg I think there was a case where `children` was coming out as `undefined`. Basically, just a safety measure in case the anchor had no children.